readme.md

```diff
@@ -923,6 +923,77 @@ const HelloBundle = dynamic({
 export default () => <HelloBundle title="Dynamic Bundle" />
 ```
 
+### Custom `<App>`
+
+<p><details>
+  <summary><b>Examples</b></summary>
+  <ul><li><a href="./examples/layout-component">Using `_app.js` for layout</a></li></ul>
+  <ul><li><a href="./examples/componentdidcatch">Using `_app.js` to override `componentDidCatch`</a></li></ul>
+</details></p>
+
+Next.js uses the `App` component to initialize pages. This can be used for:
+
+- Persisting layout between page changes
+- Keeping state when navigating pages
+- Custom error handling using `componentDidCatch`
+- Handling of calling **page** `Component`s `getInitialProps`
+
+To override the default `_app.js` behaviour you can create a file `./pages/_app.js`, where you can extend the `App` class:
```

Comment:
To override, create a file in `./pages/_app.js` and override the `App` class as shown below: